clients/elrond/interface.go

```diff
@@ -22,3 +23,14 @@ type NonceTransactionsHandler interface {
 	SendTransaction(ctx context.Context, tx *data.Transaction) (string, error)
 	Close() error
 }
+
+// elrondClientDataGetter defines the interface able to handle get requests for Elrond blockchain
+type DataGetter interface {
```

Comment:
If you move the mappers in a new package:
you will define this interface there and can use only a subset of the methods. Something like:
```GO
// DataGetter defines the interface able to handle get requests for Elrond blockchain
type DataGetter interface {
    GetTokenIdForErc20Address(ctx context.Context, erc20Address []byte) ([][]byte, error)
    GetERC20AddressForTokenId(ctx context.Context, tokenId []byte) ([][]byte, error)
    IsInterfaceNil() bool
}
```

Also, broken comment for this interface